### DISCUSSION TITLE
[New Feature] Force reinforcements to appear at given waypoint

### DISCUSSION THREAD

**Discussion by Crimsonum** [Vinifera-Developers/Vinifera#1322] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

The game has two ways for spawning reinforcements, trigger actions no. 7 and 80.

Action 7 spawns the units outside the visible map area, along the map's edge. From there they will move into the battlefield according to the team's script. However, sometimes the logic that determines where exactly the units will spawn is inadequate, and for that reason you have action 80.

Action 80 spawns the units directly at a specific waypoint, determined by the action waypoint parameter. However, if this waypoint exists outside the map's _visible area_ (`LocalSize`), the units will instead spawn at the edge of the visible area.

In other words, there is no way to spawn units in a specific waypoint outside the visible map area.

**Comment by JoyfulShush** [comment]
In order to be backwards compatible and not break maps, it might be nice to introduce a new trigger action that behaves similarly to action 80, but also respects waypoints that are outside of map bounds.
